docs/extensibility/README.md

```diff
@@ -0,0 +1,15 @@
+# Extensibility
+
+## Overview
+
+Extensibility in Busola allows you to create a UI page for your Custom Resource Defintion (CRD). You can create it at a cluster or Namespace level. To configure the UI page of your CRD, you can add navigation nodes, lists, and details pages, as well as create and edit forms. You can also add [display](display-widgets.md) and [form](form-widgets.md) to have a graphical representation of the elements.
+
+## Get started
```

Comment:
I think everywhere I've seen it, the section is always called "Getting started"... unless it's some kind of SAP standard?

Reply:
Both are perfectly fine. I used the one that is on [our webpage](https://kyma-project.io/docs/kyma/latest/02-get-started/#documentation-content), but I can change it.